### PR TITLE
Docs: improve LogQL section

### PR DESCRIPTION
**Pull request by KMiller-Grafana**
- Add a subsection for examples.  The section will be populated with more examples over time.
- Fix typos and wording

### CODE REVIEW

File: docs/sources/logql/query_examples.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        |regexp "(^(?P<ip>\\S+ {1,2}){10})"
+        | line_format "IP = {{.ip}}\tUSER = {{.user}}"
+    ```
+   
+- Get successful logins from Linux `/var/log/secure`
+
+    ```logql
+    {job="security"}
+        != "grafana_com"
+        |~ "session opened"
```
Comment (sandeepsukhani) [inline docs/sources/logql/query_examples.md:47]: There is no regex pattern so we can just do contains filter
```suggestion
        |= "session opened"
```

File: docs/sources/logql/query_examples.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Examples that aid in security evaluation
+
+- Extract the user and IP address of failed logins from Linux `/var/log/secure`
+
+    ```logql
+    {job="security"} 
+        |~ "Invalid user.*"
+        |regexp "(^(?P<user>\\S+ {1,2}){8})"
+        |regexp "(^(?P<ip>\\S+ {1,2}){10})"
```
Comment (sandeepsukhani) [inline docs/sources/logql/query_examples.md:38]: Just a nit: I like putting a space after the pipe. You have added it for some of them. I think we should add them here too?
```suggestion
        | regexp "(^(?P<user>\\S+ {1,2}){8})"
        | regexp "(^(?P<ip>\\S+ {1,2}){10})"
```

File: docs/sources/logql/query_examples.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    ```
+   
+- Get successful logins from Linux `/var/log/secure`
+
+    ```logql
+    {job="security"}
+        != "grafana_com"
+        |~ "session opened"
+        != "sudo: "
+        |regexp "(^(?P<user>\\S+ {1,2}){11})"
```
Comment (sandeepsukhani) [inline docs/sources/logql/query_examples.md:49]: Same as above

**Review by sandeepsukhani — COMMENTED** [review]
I have added some minor nits. Other than that it LGTM!
Do you think we should have sample input/output as well for better understanding?

**Review by owen-d — APPROVED** [review]

**Pull request merged**